messages/de/app.php

```diff
@@ -18,13 +18,13 @@
     'Bumped {time}' => 'Gebumpt um {time}',
     'Bumped recently' => 'Kürzlich gebumpt',
     'Cancel' => 'Abbrechen',
-    'Categorize your server by keywords (e.g. game title)' => 'Kategorisiere deinen Server mit Stichwörtern (z.B. Gaming, Anime, etc.)',
+    'Categorize your server by keywords (e.g. game title)' => 'Kategorisiere deinen Server mit Stichworten (z.B. Spielname)',
```

Comment:
I'd suggest "**Spieletitel**" instead of "Spielename".

Reply:
That does fit better - just couldn't get on the word.